src/components/bcros/Divider.vue

```diff
@@ -0,0 +1,3 @@
+<template>
+  <span class="mt-[0.1rem] mx-[0.5rem] h-4 border-l border-gray-600" />
+</template>
```

Comment:
1. Isn't there already a common component for this? (I don't know, but I imagine you're not the first to need a divider).
2. IMO, components should have zero whitespace around them, so that their parent can style them according to the paren't requirements. So, I recommend moving the top and x margins from here to the parent.

Reply:
We used NuxtUI's `<UDivider orientation="vertical">` for this vertical dividers. But this is not an inline element and does not work well inside a `<span>`.  We currently put it in a flexbox `<div class="flex flex-row">`; the filing subtitle will look like this (screenshot below) when the window is not wide enough:
<img width="649" alt="image" src="https://github.com/user-attachments/assets/32761759-3b13-4133-a12b-bad53db0fafc"> 

To fix this style issue, I tried `<div class="flex flex-wrap">`. But it didn't work very well for text wrapping. All the text to the right of the divider will be moved to the next row when it can't fit into the current row...

There might be a walkaround solution if I want to keep using UDivider. But I think the easiest way is to make our own inline divider. Just like in the Filing-UI, you are using `<span class="vert-pipe" />`.

Reply:
For 2, that is a good idea. I will move the mt and mx to the parent.

Reply:
Looks good. Resolving.